lib/util/i18n.js

```diff
@@ -0,0 +1,11 @@
+'use strict'
+
+function apply (params) {
+  if (!params) return undefined
+  if ('key' in params) return params
+  return { key: params[0], props: params[1] }
```

Comment:
hmm maybe props should be params[1] || {}

Reply:
This is fine, `props` can be undefined